python/ray/tune/tests/test_multinode_sync.py

```diff
@@ -168,196 +162,6 @@ def on_step_begin(self, iteration, trials, **info):
             callbacks=[FailureInjectionCallback()],
         )
 
-    def testCheckpointSync(self):
```

Comment:
Is this test failing? I would expect it to pass (it's not using the syncer explicitly)?

Reply:
I think this test relies on syncer callback which I removed.